drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -177,20 +197,26 @@ static int spi_mcux_configure(const struct device *dev,
 	uint32_t clock_freq;
 	uint32_t word_size;
 
-	if (spi_context_configured(&data->ctx, spi_cfg)) {
+	pm_policy_device_power_lock_get(data->dev);
```

Comment:
I actually don't think you even need to call get/put in the configure function either, because this function is static and only called by transceive, which you are already locking from there. So ignore my other comment about the error handling because you don't even need to put all this new code in this function for the power lock get/put.